docs/CHANGELOG.md

```diff
@@ -1,145 +1,55 @@
 # Changelog
 
-## [12.0.7] - 2022-06-30
-
-<details>
-    <summary>Bits</summary>
+## [12.0.7] 📅 2022-06-30
 
 ### Added
 
-- ToolbarMessageComponent
-- ToolbarComponent default message for state with no items
+- `@nova-ui/bits` | _NUI-6180_ | _ToolbarComponent_ now has default message for state with no items (implemented using new __nui-toolbar-message__ component which can also be used by consumers explicitly inside __nui-toolbar__)
 
-</details>
+### Bugfix
 
-<details>
-    <summary>Dashboards</summary>
+- `@nova-ui/dashboards` | _DashboardUnitConversionPipe_ now takes unit type to determine nuit number base
 
-### Changed
+## [12.0.6] 📅 2022-05-25
 
-- DashboardUnitConversionPipe takes unit type to determine nuit number base
+### Bugfix
 
-</details>
+- `@nova-ui/dashboards` | Fixed bug, when "Explore this data" in the widget's header icon does not open a defined URL in some cases.
 
-## [12.0.0]
+## [12.0.5] 📅 2022-05-04
 
-This section is going to be added. (Leaving the section below for reuse)
+### Bugfix
 
-## [11.0.0] - 2021-11-21
```

Comment:
I think it would be still useful to keep changes for version 11, maybe put them to a different file instead of removing them?